test/composables/useTheme.test.ts

```
@@ -36,6 +36,10 @@ describe('composition API', () => {
         defaultBaseUrl: 'https://app.local',
         getServers: ({ method, path, operation }) => (`https://app.local/${method}${path}`),
       },
+      server: {
+        allowCustomServers: true,
+        getServers: ({ method, path, operation }) => [`https://app.local/${method}${path}`],
```

ESLint

[error] 'operation' is defined but never used. Allowed unused args must match /^_/u.

(unused-imports/no-unused-vars)